### PR TITLE
Remove runc replace directive in go.mod

### PR DESCRIPTION
**Pull request by chrisdoherty4**
We patched a security vuln in our runc dep using a replace directive because dependencies couldn't build with a proper upgrade.

Our deps have since updated meaning the replace directive isn't required and we can just upgrade.

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -210,7 +225,7 @@ replace (
 	// opencontainers/runc patched a CVE in v1.1.5.
 	// Various dependencies both direct and indirect are yet to upgrade runc. All seem to work
 	// with v1.1.5 so we're forcing it with replace.
-	github.com/opencontainers/runc => github.com/opencontainers/runc v1.1.5
+	//github.com/opencontainers/runc => github.com/opencontainers/runc v1.1.5
```
Comment (g-gaston) [inline go.mod:228]: Delete this block?

Comment (chrisdoherty4) [reply go.mod:228]: Oh, thanks. That's what happens when you leave something simple for several days :D.

Comment (chrisdoherty4) [reply go.mod]: Fixed.

**Review by g-gaston — COMMENTED** [review]
/lgtm

**Pull request merged**